services/core-api/app/api/now_applications/resources/now_application_status_resource.py

```diff
@@ -143,7 +145,13 @@ def put(self, application_guid):
 
             #TODO: Documents / CRR
             # Update NoW application and save status
-            now_application_identity.now_application.status_updated_date = datetime.today()
+            if now_application_status_code == 'REJ':
```

Comment:
We don't need to worry about "REI" Rejected-Initial status code at all, do we?

Reply:
As far as I am aware it is just on the process permit page where the permit can be approved, Rejected, or Withdrawn(also REJ). I didnt see anything in the ticket about REI.

Reply:
I think REI is getting tossed.

Reply:
I will likely need to have "No Permit Required" status end to progress - Will address in my coming PR.